src/tests/test_api.py

```diff
@@ -140,33 +140,25 @@ def test_fetch_genome_dataset_unreleased(self, multi_dbs):
         assert test[0][1].is_current == False
 
     def test_fetch_genome_info(self, multi_dbs):
-        conn = GenomeAdaptor(metadata_uri=multi_dbs['ensembl_metadata'].dbc.url,
-                             taxonomy_uri=multi_dbs['ncbi_taxonomy'].dbc.url)
-        test = conn.fetch_genome_datasets(unreleased_datasets=True)
-        print(test)
-        assert test[0][1].release_id is None
-        assert test[0][1].is_current is False
-
-    def test_fetch_genome_info_unreleased(self, multi_dbs):
         conn = GenomeAdaptor(metadata_uri=multi_dbs['ensembl_metadata'].dbc.url,
                              taxonomy_uri=multi_dbs['ncbi_taxonomy'].dbc.url)
         test = conn.fetch_genome_datasets(unreleased_datasets=True)
         assert test[0][1].release_id is None
-        assert test[0][1].is_current == False
-
-    def test_fetch_genome_info(self, multi_dbs):
-        conn = GenomeAdaptor(metadata_uri=multi_dbs['ensembl_metadata'].dbc.url,
-                             taxonomy_uri=multi_dbs['ncbi_taxonomy'].dbc.url)
-        test = conn.fetch_genomes_info()
-        result = next(test)[0]
-        assert 'genome' in result
-        assert 'datasets' in result
-
-    def test_fetch_genome_info_genome_uuid(self, multi_dbs):
-        uuid = 'a7335667-93e7-11ec-a39d-005056b38ce3'
-        conn = GenomeAdaptor(metadata_uri=multi_dbs['ensembl_metadata'].dbc.url,
-                             taxonomy_uri=multi_dbs['ncbi_taxonomy'].dbc.url)
-        test = conn.fetch_genomes_info(genome_uuid=uuid)
-        result = next(test)[0]
-        assert result['genome'][0].genome_uuid == uuid
-        assert result['datasets'][0][0].genome_uuid == uuid
+        assert test[0][1].is_current == 0
+
+    #Duplicate
```

Comment:
not sure genome_inf is covered in any other test cases

Reply:
I left in the singular test. Next is not alwalys appropriate and the "unreleased_only" is silly as the table won't be created until the genome is released. Checking for an absence is better, but it is outside the scope of this PR.